assets/dev/scss/frontend/widgets/video.scss

```diff
@@ -47,3 +47,11 @@
 		}
 	}
 }
+
+.e-container {
+
+	> .elementor-widget-video {
+		// Fix - Video widget has an undefined height on mobile using 'flex column wrap' inside the flex container (ED-7604).
```

Comment:
I have added the `>` symbol to make sure we are only targeting the direct child. 

This could be relevant e.g. for the nested tab widget, where we can have a multi levels of containers and widgets which need to be handled seperately.